clang/test/3C/canwrite_constraints.h

```diff
@@ -71,6 +71,13 @@ void unwritable_func_with_itype_and_bounds(int *p
 // expected-warning@+1 {{Declaration in non-writable file}}
 _Itype_for_any(T) void my_generic_function(void *p : itype(_Ptr<T>));
 
+// expected-warning@+1 {{Declaration in non-writable file}}
+_Itype_for_any(T) void* my_generic_return(void) : itype(_Ptr<T>);
+
+void unwritable_type_return() { 
```

Comment:
The name `unwritable_type_argument` referred to insertion of a "type argument" `<int>` corresponding to the type parameter `T`, not the fact that `T` is used in a parameter type of `my_generic_function` rather than the return type.  So the name `unwritable_type_return` doesn't make much sense to me.  Instead, maybe just combine this function body with `unwritable_type_argument`?

Reply:
sure thing